.github/scripts/js/ci.js

```diff
@@ -801,26 +964,21 @@ module.exports.createReleaseIssueForMilestone = async ({ github, context, core }
 };
 
 /**
- * Find the recent milestone and it's "release" issue. Create new comment and
- * start build-and-test_release workflow with the ID of the created comment.
+ * Find the recent milestone related to the Git ref.
  *
  * @param {object} inputs
  * @param {object} inputs.github - A pre-authenticated octokit/rest.js client with pagination plugins.
  * @param {object} inputs.context - An object containing the context of the workflow run.
  * @param {object} inputs.core - A reference to the '@actions/core' package.
- * @returns {Promise<void>}
+ * @param {object} inputs.gitRefInfo - A Git ref info.
+ * @returns {object} - A milestone or an error message.
  */
-const startBuildAndTestWorkflow = async ({ github, context, core }) => {
-  const github_ref = context.ref;
-
-  // TODO Temporarily no comment for release-* branches.
-
-
-  // Find 10 recently created milestones.
+const findMilestoneForGitRef = async ({ github, context, core, gitRefInfo }) => {
+  // Find first 25 recently created milestones.
   const query = `
     query($owner:String!, $name:String!) {
       repository(owner:$owner, name:$name){
-        milestones(first:10, orderBy:{field:CREATED_AT, direction:DESC}, states:[OPEN]) {
+        milestones(first:25, orderBy:{field:CREATED_AT, direction:DESC}, states:[OPEN]) {
```

Comment:
Why do we limit the number of milestones? Could we request 1k milestones at once?

Reply:
If we have 26 open milestones, then we'll have silent bug here. Maybe it's unlikely condition, but still easy to avoid that bug.

Reply:
The limit for the graphql query is 100 per request. I can set 100, why not.

Reply:
Done